tools/site-scan.rb

```diff
@@ -146,7 +146,7 @@ def parse(id, site, name)
   end
   # THIRD: see if an image has been uploaded
   if IMAGE_DIR
-    data[:image] = Dir[File.join(IMAGE_DIR, "#{id}.*")].
+    data[:image] = Dir[File.join(IMAGE_DIR, "#{id}*.{svg,eps,ai,pdf}")].
```

Comment:
Is there a documented reason to scan for only those file types?  Why not include .png as a catch-all (Which is all some projects used to have).

Reply:
The reason why I added this, is because [logos/about.html](http://www.apache.org/logos/about.html) states that `svg`, `eps`, `ai` and `pdf` file types are accepted. Checking for PNGs would result in 'Yeah, the logo is present!', however the logo won't be propagated to a.o/logos which IMO can confuse some people as well.

Side note: a lot of originals are already scalable types (check [here](https://svn.apache.org/repos/asf/comdev/project-logos/originals/))